packages/paste-website/src/pages/components/time-picker/index.mdx

```diff
@@ -0,0 +1,608 @@
+      <HelpText id="foo_text">Please select a time.</HelpText>
+    </>
+  );
+};
+```
+
+#### TimePicker Props
+
+All the valid HTML attributes for `input` (except `type`) are supported, including the following props:
+
+| Prop      | Type                                     | Description                                                          | Default   |
+| --------- | ---------------------------------------- | -------------------------------------------------------------------- | --------- |
+| id?       | string                                   | Sets the id of the time picker. Should match the htmlFor of `Label`. | null      |
+| name?     | string                                   | Sets the name of the time picker.                                    | null      |
+| value?    | string                                   | Sets the value of the time picker.                                   | null      |
+| disabled? | boolean                                  | Disables the time picker.                                            | false     |
+| readOnly? | boolean                                  | Sets the time picker to readonly.                                    | false     |
+| required? | boolean                                  | Sets the time picker as required.                                    | false     |
+| hasError? | boolean                                  | Sets the time picker to an error state.                              | false     |
+| min?      | string                                   | Sets the earliest valid time value.                                  | null      |
+| max?      | string                                   | Sets the last valid time value.                                      | null      |
+| onChange? | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| onFocus?  | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| onBlur?   | `(event: React.MouseEvent<HTMLElement>)` |                                                                      | null      |
+| variant?  | 'default', 'inverse'                     |                                                                      | 'default' |
+
+#### formatReturnTime() Props
+
+| Prop       | Type   | Description                                                                                                           | Default |
+| ---------- | ------ | --------------------------------------------------------------------------------------------------------------------- | ------- |
+| timeValue  | string | Time value to be formatted. Must be in <inlineCode>HH:mm</inlineCode> format.                                         | null    |
+| timeFormat | string | Speficy the format in which to return the timeValue. Formats come from the <inlineCode>date-fns</inlineCode> library. | null    |
```

Comment:
```suggestion
| timeFormat | string | Specify the format in which to return the timeValue. Formats come from the <inlineCode>date-fns</inlineCode> library. | null    |
```

Reply:
Good catch 😂 fixed here and in date picker